api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -2220,6 +2220,21 @@
         </field>
       </fields>
       <codeSegments>
+        <codeSegment>
+            <version>4.0.0+</version>
+            <code>
+            <![CDATA[
+    @Deprecated
```

Comment:
So these are new methods being added in a deprecated state? That's unusual.

There should be a Javadoc comment about them of some kind, perhaps just indicating that they should not be used at all.

And if they're deprecated because they're only for internal use, that's not really what deprecated is typically used for.

Reply:
I've added a bit of javadoc.
Those methods are not new, they are not explicitly added, because the switch above from `<version>4.0.0+</version>` to `<version>4.0.0</version>` effectively does not generate them anymore.  So I added them for backward compatibility as no-op.